pkg/lvm/lvm_util.go

```diff
@@ -507,6 +536,94 @@ func ListLVMVolumeGroup() ([]apis.VolumeGroup, error) {
 	return decodeVgsJSON(output)
 }
 
+/*
```

Comment:
For all the below functions, augment the function description comments with more information and example command inputs and outputs.

Reply:
Done.